python/lsst/ts/ofc/ofc_data/ofc_data.py

```diff
@@ -100,8 +100,14 @@ class OFCData(BaseOFCData):
         If input `config_dir` does not exists.
     """
 
-    def __init__(self, name=None, config_dir=None, log=None, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(
+        self,
+        name: str | None = None,
+        config_dir: str | None = None,
+        log: logging.Logger | None = None,
+        **kwargs: object,
+    ) -> None:
+        super().__init__()
```

Comment:
I think the original code is:

```python
super().__init__(**kwargs)
```

It passes the keyword arguments to the parent class.

Reply:
In this case, you can ignore the mypy check by following:
https://stackoverflow.com/questions/49220022/how-can-mypy-ignore-a-single-line-in-a-source-file